packages/storybook/stories/MaintenanceBanner.stories.jsx

```diff
@@ -33,10 +47,10 @@ const defaultArgs = {
   content:
     'We’re working on VA.gov right now. If you have trouble signing in or using tools, check back after we’re finished. Thank you for your patience.',
   warnContent:
-    'We’ll be doing some work on VA.gov. The maintenance will last 1 hour. During that time, you won’t be able to sign in or use tools.',
+    'We’ll be doing some work on VA.gov. The maintenance will last X hours. During that time, you won’t be able to sign in or use tools.',
   startsAt: new Date(Date.now()),
-  expiresAt: addHours(Date.now(), 1),
-  warnStartsAt: subMinutes(Date.now(), 30),
+  expiresAt: addHoursToDate(new Date(Date.now()), 2),
```

Comment:
Would [`.setHours()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setHours) be useful here? We're doing something similar in the tests:

https://github.com/department-of-veterans-affairs/component-library/blob/cfc396eeed001fccc804e56e17a82bd55499e6f7/packages/react-components/src/components/MaintenanceBanner/MaintenanceBanner.unit.spec.jsx#L11-L14

Reply:
@bkjohnson Updated this PR to remove the additional functions and leverage `.setHours` instead